lib/batch.js

```diff
@@ -14,57 +14,63 @@ function inject(server, options) {
 }
 
 // called in a route as `batch(req, map)`, where map is an Object
-// mapping property names to routes.
+// mapping routes to fields that should be extracted from their response.
+// Mapping a route to `true` will include all of its returned fields,
+// while mapping it to an array will include only the fields named
+// in the array.  For example:
 //
-// Ex: batch(req, { email: '/v1/email', avatar: '/v1/avatar' }).done(reply)
+//   batch(req, {
+//     // Include all fields returned by /v1/email
+//     '/v1/email': true
+//     // Include only two of the fields returned by /v1/avatar
+//     '/v1/avatar': ['avatar', 'avatarDefault']
+//   }).done(reply)
```

Comment:
This is a significant change to how `batch` works; previously it could only fetch a single field from each route that it calls, and I've changed it so that it basically flattens all the fields into a single object.